src/ext.js

```diff
@@ -333,7 +345,7 @@ export default function ext({ translator }) {
                         {
                           component: 'icon-item',
                           icon: 'align_left',
-                          value: 'left',
+                          value: 'flex-start',
```

Comment:
I know I changed it into flex-start and flex-end, but not so sure if we want it like that. If we would like to change the css in the future we would need to do logic based on these values instead of left and right

Reply:
hmm true. I can change back